Documentation for CGroup toggle in toolkit/docs/formats/imageconfig.md to generate Mariner images with cgroupv2 (#4908)
microsoft/CBL-Mariner: toolkit/docs/formats/imageconfig.md

This will instruct init (systemd) to set the configured mode on boot.  The `force_enforcing` option will set enforcing in the config and also add `enforcing=1` in the kernel command line,
which is a higher precedent than the config file. This ensures SELinux boots in enforcing even if the /etc/selinux/config was altered.

The version for CGroup in Mariner images can be enabled by using the `CGroup` key with value containing which version to use on boot. The value that can be chosen is either `version_one` or `version_two`. 

> The `version_two` value will set the cgroupv2 to be used in Mariner by setting the config value `systemd.unified_cgroup_hierarchy=1` in /boot/systemd.cfg. The value `version_one` or no value set will keep

Reviewer: ```suggestion
The `version_two` value will set the cgroupv2 to be used in Mariner by setting the config value `systemd.unified_cgroup_hierarchy=1` in the default kernel command line. The value `version_one` or no value set will keep
```
nit - not sure if the exact detail for /boot/systemd.cfg should be explained. Ultimately, the change adds `systemd.unified_cgroup_hierarchy=1` into the kernel command line, which today is implemented by grub.cfg's import of /boot/systemd.cfg to set an env var, which is used in the actual kernel command line arguments. But the overall expected behavior of this is to have `systemd.unified_cgroup_hierarchy=1` present in the kernel cmdline when `version_two` value is used.

> cgroupv1 (current default) to be enabled on boot. This KernelCommandLine option can be used to quickly generate Mariner images (including marketplace) with cgroupv2 enabled to fulfill customer needs.

Reviewer: ```suggestion
cgroupv1 (current default) to be enabled on boot.
```
nit - I think this sentence is not necessary to state since the purpose of `KernelCommandLine` is already stated earlier.

> For more information about cgroups, see [About cgroupv2](https://kubernetes.io/docs/concepts/architecture/cgroups/).

Reviewer: ```suggestion
For more information about cgroupsv2 with Kubernetes, see [About cgroupv2](https://kubernetes.io/docs/concepts/architecture/cgroups/).
```
